### PR TITLE
Tracking changes needed to personalize the manifests.

### PR DESCRIPTION
**Pull request by sbose78**
( do not merge ) Tracking changes needed to personalize the manifests.

### CODE REVIEW

File: setup.sh [hunk 1]
```diff
@@ -40,9 +40,9 @@ if [ ! -f "${FILENAME}" ]; then
     exit 1
 fi
 
-sed $SED_OPTIONS "s|REPLACE_IMAGE|${IMAGE_REPO}|g" deploy/**/*.yaml pipelines/**/*.yaml
-sed $SED_OPTIONS "s|PULL_SECRET_NAME|${PULL_SECRET_NAME}|g" pipelines/serviceaccount/serviceaccount.yaml
-sed $SED_OPTIONS "s|QUAYIO_USERNAME|${QUAYIO_USERNAME}|g" pipelines/bootstrap.sh
-sed $SED_OPTIONS "s|DOCKER_CONFIG_NAME|${DOCKER_CONFIG_NAME}|g" pipelines/bootstrap.sh
-sed $SED_OPTIONS "s|GITHUB_REPO|${GITHUB_REPO}|g" pipelines/eventlisteners/cicd-event-listener.yaml
-sed $SED_OPTIONS "s|GITHUB_STAGE_REPO|${GITHUB_STAGE_REPO}|g" pipelines/eventlisteners/cicd-event-listener.yaml
+sed -i "" "s|REPLACE_IMAGE|${IMAGE_REPO}|g" deploy/**/*.yaml pipelines/**/*.yaml
```
Comment (bigkevmcd) [inline setup.sh:43]: Can you revert and try with the `$SED_OPTIONS` instead?

**Review by bigkevmcd — CHANGES_REQUESTED** [review]
The sed changes are for MacOS.

And it looks like this has had the changes applied for your repo?

**Comment by sbose78** [comment]
Yes, this is just a "for our reference" PR. This isn't to be merged. :)

File: pipelines/cd/task-deploy-using-kubectl-without-replacement.yaml [hunk 2]
```diff
@@ -0,0 +1,28 @@
+apiVersion: tekton.dev/v1alpha1
+kind: Task
+metadata:
+  name: deploy-using-kubectl-without-replacement
```
Comment (sbose78) [inline pipelines/cd/task-deploy-using-kubectl-without-replacement.yaml:4]: Have a look at this, Kevin.

**Pull request closed without merging**